pkg/engine/jmespath/functions.go

```diff
@@ -210,6 +212,15 @@ func getFunctions() []*gojmespath.FunctionEntry {
 			},
 			Handler: jpBase64Encode,
 		},
+		{
+			Name: timeSince,
```

Comment:
Just an idea:
Is it possible to have a function with the same name but 2 arguments?
So:
```
"time_since('2021-01-02T15:04:05-07:00', '2021-01-10T03:14:05-07:00')
```
and
```
time_since('Mon Jan _2 15:04:05 MST 2006', 'Mon Jan 02 15:04:05 MST 2021', 'Mon Jan 10 03:14:16 MST 2021')
```
are both valid calls

Reply:
Hey @JimBugwadia, apart from this^, the rest is complete.

Reply:
Ok. @ojhaarjun1 - if there is no way to support a variable number of args (1, 2, or 3) lets proceed with the 3 argument version and complete.